WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooCarrierPackagesSelectionView.swift

```diff
@@ -101,11 +101,20 @@ struct WooCarrierPackagesSelectionView: View {
                            tabItemContentHorizontalPadding: Constants.tabItemContentHorizontalPadding,
                            tabItemContentVerticalPadding: Constants.tabItemContentVerticalPadding)
             }
-            if viewModel.isLoadingPackages {
+            // Show extra loading indicator in case there are no packages
+            else if viewModel.isLoadingPackages {
                 ProgressView()
                     .progressViewStyle(.circular)
                     .padding()
             }
+            else {
+                Button {
+                    viewModel.loadPackages()
+                } label: {
+                    Image(systemName: "arrow.trianglehead.counterclockwise")
+                }
+                .padding()
```

Comment:
Showing loading indicator only if we did not have any data before, and showing the refresh icon if we got no data from backend (so we can load packages manually since we do not have the list and pull to refresh in that case)